script.js

```diff
@@ -0,0 +1,34 @@
+// Set Count Down date
+let countDownDate = new Date("Jan 1, 2024 00:00:00").getTime();
+
+// Update count down timer every 1 second
```

Comment:
- [Optional] It is often considered a good practice to remove comments before merging into the main branch so you might want to remove inline comment then merge.

Reply:
OK Sir thank you